charts/prometheus-blackbox-exporter/templates/selfservicemonitor.yaml

```diff
@@ -16,6 +16,9 @@ spec:
     interval: {{ .Values.serviceMonitor.selfMonitor.interval }}
     scrapeTimeout: {{ .Values.serviceMonitor.selfMonitor.scrapeTimeout }}
     scheme: http
+    {{- with .Values.serviceMonitor.selfMonitor.port }}
```

Comment:
@monotek what should be the value for port key if it is defined in values file ? I was uncertain about the value so I defined it this way if user wishes to define port in `values.yaml` it will automatically get included in `selfservicemonitor.yaml` file

Reply:
Could also be a commented out one. Just so users know its configurable at all.

Reply:
Please checkout the latest commit.